Nos/Models/RelaySubscription.swift

```diff
@@ -42,7 +42,7 @@ struct RelaySubscription: Identifiable {
     ) {
         self.filter = filter
         self.relayAddress = relayAddress
-        self.id = filter.id + "-" + relayAddress.absoluteString
+        self.id = (filter.id + "-" + relayAddress.absoluteString).data(using: .utf8)?.sha256 ?? "error"
```

Comment:
I'm glad you spotted it. Do you mind adding a comment in the code linking to this PR? Just in case in the future we wonder the same thing.

Reply:
Good idea! Done!